src/pages/discover/playlist/index.tsx

```
@@ -3,18 +3,34 @@
 import styled from 'styled-components'
 
 import { Cancel } from '@/assets/icons'
-import PlaylistData from '@/pages/discover/playlistData.json'
+import { usePlaylistDetail } from '@/entities/playlist'
+import { getGenreLabel } from '@/shared/lib'
 import { flexColCenter } from '@/shared/styles/mixins'
-import { Header, Link, SvgButton } from '@/shared/ui'
+import { Error, Header, Link, Loading, SvgButton } from '@/shared/ui'
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)